### PR TITLE
jigasi: autoscaler sidecar support

### PR DESCRIPTION
**Pull request by aaronkvanmeerten**

### CODE REVIEW

**Review by damencho — APPROVED** [review]

File: jigasi/rootfs/etc/cont-init.d/10-config [hunk 1]
```diff
@@ -13,6 +13,41 @@ if [[ "$JIGASI_XMPP_PASSWORD" == "$OLD_JIGASI_XMPP_PASSWORD" ]]; then
     exit 1
 fi
 
+# Local IP for autoscaler sidecar reporting
+export LOCAL_ADDRESS=$(ip route get 1 | grep -oP '(?<=src ).*' | awk '{ print $1 '})
```
Comment (saghul) [inline jigasi/rootfs/etc/cont-init.d/10-config:17]: Why is this necessary?

File: jigasi/Dockerfile [hunk 2]
```diff
@@ -11,7 +11,7 @@ LABEL org.opencontainers.image.documentation="https://jitsi.github.io/handbook/"
 ENV GOOGLE_APPLICATION_CREDENTIALS /config/key.json
 
 RUN apt-dpkg-wrap apt-get update && \
-    apt-dpkg-wrap apt-get install -y jigasi jq && \
+    apt-dpkg-wrap apt-get install -y jigasi jq jitsi-autoscaler-sidecar && \
```
Comment (saghul) [inline jigasi/Dockerfile:14]: Let's install Node from the NodeSource repo please.

Comment (aaronkvanmeerten) [reply jigasi/Dockerfile:14]: This happens in the base-java image now.
